webapp/components/fiatBalance.tsx

```diff
@@ -37,7 +37,8 @@ const RenderFiatBalanceUnsafe = function ({
 
   const stringBalance = formatUnits(balance, token.decimals)
 
-  const price = data?.[token.symbol.toUpperCase()] ?? '0'
+  const price =
+    data?.[(token.extensions?.priceSymbol ?? token.symbol).toUpperCase()] ?? '0'
```

Comment:
This line of code is a bit complex to read at first glance. I would suggest breaking it down into at least two steps, something like this below. Thoughts?
```suggestion
const priceKey = (token.extensions?.priceSymbol ?? token.symbol).toUpperCase()
const price = data?.[priceKey] ?? '0'
```

Reply:
Simplified here [276dc21](https://github.com/hemilabs/ui-monorepo/pull/918/commits/276dc21904586e1b381d3768ba351af747ba0e9f)